src/constants/exploreEndpointsPages.ts

```diff
@@ -461,22 +489,59 @@ export const EXPLORE_ENDPOINTS_PAGES_HORIZON: ExploreEndpointsPagesProps = {
         {
           route: Routes.EXPLORE_ENDPOINTS_PAYMENTS,
           label: "All Payments",
-          form: undefined,
+          form: {
+            docsUrl:
+              "https://developers.stellar.org/network/horizon/resources/list-all-payments",
+            docsLabel: "payments",
+            requestMethod: "GET",
```

Comment:
i like using enum for some of the repeated constants. just a thought! def don't have to make a change in this pr.

Reply:
We're using "GET" as a string here because it's coming from the type (`requestMethod: "GET" | "POST";`). I find it more convenient to use it this way in this case and also easier to validate.